bin/runtime/src/lib.rs

```diff
@@ -518,9 +519,13 @@ impl_runtime_apis! {
         }
     }
 
-    impl primitives::AlephApi<Block> for Runtime {
-        fn authorities() -> Vec<AlephId> {
-            Aleph::authorities()
+    impl primitives::AlephApi<Block, AlephId, BlockNumber> for Runtime {
+        fn current_session() -> Option<AuthoritySession<AlephId,BlockNumber>> {
```

Comment:
There are some styling issues, did you run `local-pipeline`?

Reply:
Should be fixed now